test/appium/tests/test_chat_management.py

```diff
@@ -0,0 +1,75 @@
+import time
+
+import pytest
+
+from tests import transaction_users
+from tests.base_test_case import MultipleDeviceTestCase
+from views.sign_in_view import SignInView
+
+
+@pytest.mark.all
+@pytest.mark.chat_management
+class TestChatManagement(MultipleDeviceTestCase):
+
+    @pytest.mark.testrail_case_id(3412)
+    def test_delete_1_1_chat(self):
+        self.senders['user_1'] = transaction_users['G_USER']
+        self.senders['user_2'] = transaction_users['H_USER']
```

Comment:
Let's call them in one way - like here https://github.com/status-im/status-react/pull/4014/files#diff-0ffa44627c9105584ecb93a94e163c91R193
I mean we should have either users 1,2,... or a,b, etc. to not mess them

Reply:
Done